randomize/randomize.go

```diff
@@ -52,7 +52,7 @@ var (
 	rgxValidTime     = regexp.MustCompile(`[2-9]+`)
 
 	validatedTypes = []string{
-		"inet", "line", "uuid", "interval", "mediumint",
+		"inet", "line", "uuid", "uniqueidentifier", "interval", "mediumint",
```

Comment:
I remain unconvinced that there needs to be any separation for this. It's treated the same by randomize.go, and there are no special cases for the field that I can see in the code you've introduced.

There's two cases here:
1. If I have a uniqueidentifier in SQL Server field it will break in the generated tests since the uuid package is being used to generate a V4 UUID and not a GUID per se, then we need to fix this piece.
2. There's no special cases, and we can easily coerce this into the type uuid just to have less special cases (there's obviously plenty already from the introduction of mssql :) Keep in mind the driver still gets to determine what real type a "uuid" produces, so it shouldn't matter so much unless you want randomize.go to treat it differently.

I'm willing to be convinced, but I don't see anything convincing yet :)